netlify.toml

```diff
@@ -25,3 +25,9 @@
     to = "http://main.gateway-api.sigs.k8s.io/:splat"
     status = 301
     force = true
+
+[[redirects]]
+    from = "/geps/gep-1426"
+    to = "/geps/gep-1294"
+    status = 301
```

Comment:
Deploy preview shows this working correctly.

See https://deploy-preview-3359--kubernetes-sigs-gateway-api.netlify.app/geps/gep-1426/

Reply:
wonder why we need that? for previous versions of the website?

Reply:
In case there are permalinks anywhere that point to the Service Mesh GEP (which used to incorrectly be at that location). This will redirect people to the now corrected page.